ChangeLog

```diff
@@ -1,5 +1,38 @@
 ### 3.9.1dev <- NOTE: The release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
+  (contributed by @github-actions)
+
+- Build: Mac: Keep the signing-related keychain unlocked (#2927).
```

Comment:
I thought we had documented this already...

Reply:
Will check. Maybe the PR# was not mentioned.

Reply:
Can't see anything relevant already in the ChangeLog

Reply:
I think 
> Build: Updated bundled JACK (Windows-only) to version 1.9.22 (#3031). (contributed by @ann0see)

Is it

Reply:
Ok. Removed that and some smaller fixes. Will merge now.